src/applications/appeals/10182/pages/areaOfDisagreement.js

```diff
@@ -71,7 +71,8 @@ export default {
                 formData.areaOfDisagreement[index],
               ),
             }),
-            ariaDescribedby: 'other-hint-text',
+            // index is appended to this ID in the TextWidget
+            ariaDescribedby: 'other-hint-text-',
```

Comment:
Do we need to include the `-` here, or can `addIndex` include a `_` like we have elsewhere? I don't really have a preference, but I find the mixed use of `_${index}` and `-${index}` a bit confusing.

Reply:
I could leave off the dashes in the `ariaDescribedby`. I guess I didn't think too much about using an underscore vs dashes. I tend to use dashes in IDs that are within the form components. I was following the underscore convention used within the form system.

Reply:
on top of that, `vads-u-` classnames all use dashes. I guess I lean towards using dashes over underscores in the DOM.